packages/client/src/modules/common/components/crud/FilterView.web.jsx

```diff
@@ -4,7 +4,7 @@ import { Formik } from 'formik';
 import { DebounceInput } from 'react-debounce-input';
 
 import { hasRole } from '../../../user/containers/Auth';
-import { Form, FormItem, Input, Row, Col, Button, Icon } from '../web';
+import { Form, FormItem, Input, Row, Col, Button, Icon } from '../web/index-antd';
```

Comment:
Why `index-antd.jsx` is needed at all? Why not importing `../web/ui-antd` without adding dumb file? The second question - is why are we depending on `ui-antd` here?

Reply:
I'll change import, dump file not needed. We depending on ui-antd because all CRUD cli templates needed it, in contrast whole app use bootsrap-ui and imports from/web/index.jsx hence we can't import index both for CRUD and app components because conflicts of names among those two libraries stop us.

Reply:
@Werter12 No, this is not true. The app is importing all the components from `web/index.jsx`, but `web/index.jsx` is exporting either AntD or Bootstrap components. To change this you just uncomment/comment one line in index.jsx. Your changes has broken this mechanism. Please revert your changes

Reply:
@Werter12 We cannot merge this PR until it will support all the UI's we have in the kit: AntD, Bootstrap, AntD mobile, Native Base. Please work on adding this support

Reply:
I think some features are a bit `antd` specific, so I think we could add support from other UI's in a more limited scope, meaning not having all features available. Inline editing and sorting are an example of that functionality.

Reply:
@mitjade Yes, thats fine I think. But generated code should generally work for all UIs, though with limited functionality on some UIs

Reply:
@vlasenko Yes. I agree.